docs/dictionary.rst

```diff
@@ -386,6 +388,22 @@ DOI
 
 
 
+.. _dict DRS:
+
+DRS
```

Comment:
Beyond the scope of this PR, but I'd suggested before and wonder again if this and other generated files shouldn't be in .gitignore. I started to review this before realizing I already had reviewed what this is generated from.